NetCasbin.UnitTest/Util/TestUtil.cs

```diff
@@ -59,60 +59,60 @@ internal static void TestGetFilteredGroupingPolicy(Enforcer e, int fieldIndex, L
 
         internal static void TestHasPolicy(Enforcer e, List<string> policy, bool res)
         {
-            var myRes = e.HasPolicy(policy); 
+            bool myRes = e.HasPolicy(policy);
```

Comment:
It is set at here, IDE will suggest it:
```csharp
# Types: use keywords instead of BCL types, and permit var only when the type is clear
csharp_style_var_for_built_in_types = false:suggestion
csharp_style_var_when_type_is_apparent = false:none
csharp_style_var_elsewhere = false:suggestion
dotnet_style_predefined_type_for_locals_parameters_members = true:suggestion
dotnet_style_predefined_type_for_member_access = true:suggestion
```

Reply:
Ok, vscode doesn't suggest the change. This change looks reasonable